loleaflet/src/control/Control.RowHeader.js

```diff
@@ -249,88 +247,6 @@ L.Control.RowHeader = L.Control.Header.extend({
 		ctx.fillRect(startX, endY - 1, width , this._borderWidth);
 	},
 
-	drawGroupControl: function (group) {
```

Comment:
Are there new functions for rendering/handling the "Group and outline" feature's ui ? This feature used to work well with freeze panes and I think these functions were still used.